.github/workflows/builds.yml

```diff
@@ -38,9 +38,14 @@ on:
         type: boolean
         description: 'Observer'
         required: false
-      include_without_netgo:
+      include_with_adx:
```

Comment:
I don't think we need a variable for this. It should be the default and built every time

Reply:
Good idea, implemented in implemented in https://github.com/onflow/flow-go/commit/f1e015e30d3ca9975c0bac755dc5f46e9263b3e8